docs/development/features.md

```diff
@@ -0,0 +1,151 @@
+# Rancher Desktop Features
+
+This document lists the high level Rancher Desktop features and their current status.
```

Comment:
`high-level`

Reply:
@ericpromislow , I changed to `high-level` per your suggestion.